### PR TITLE
API: rename poll_answer_id to poll_answer in post interactions vote endpoint

### PR DESCRIPTION
**Pull request by jhass**
As discussed in https://github.com/diaspora/api-documentation/pull/47

### CODE REVIEW

File: spec/integration/api/post_interactions_controller_spec.rb [hunk 1]
```diff
@@ -365,7 +365,7 @@
       post(
         api_v1_post_vote_path(@poll_post.guid),
         params: {
-          poll_answer_id: @poll_answer.id,
+          poll_answer: @poll_answer.id,
```
Comment (diaspora-code-review) [inline spec/integration/api/post_interactions_controller_spec.rb]: Layout/AlignHash: Align the elements of a hash literal if they span more than one line.

File: spec/integration/api/post_interactions_controller_spec.rb [hunk 2]
```diff
@@ -377,15 +377,15 @@
...
```
Comment (diaspora-code-review) [inline spec/integration/api/post_interactions_controller_spec.rb]: Layout/AlignHash: Align the elements of a hash literal if they span more than one line.

File: spec/integration/api/post_interactions_controller_spec.rb [hunk 3]
```diff
@@ -377,15 +377,15 @@
...
           access_token:   access_token
         }
       )
       expect(response.status).to eq(204)
       post(
         api_v1_post_vote_path(@poll_post.guid),
         params: {
-          poll_answer_id: @poll_answer.id,
+          poll_answer: @poll_answer.id,
```
Comment (diaspora-code-review) [inline spec/integration/api/post_interactions_controller_spec.rb]: Layout/AlignHash: Align the elements of a hash literal if they span more than one line.

File: spec/integration/api/post_interactions_controller_spec.rb [hunk 4]
```diff
@@ -396,7 +396,7 @@
       post(
         api_v1_post_vote_path(@poll_post.guid),
         params: {
-          poll_answer_id: -1,
+          poll_answer: -1,
```
Comment (diaspora-code-review) [inline spec/integration/api/post_interactions_controller_spec.rb]: Layout/AlignHash: Align the elements of a hash literal if they span more than one line.

File: spec/integration/api/post_interactions_controller_spec.rb [hunk 5]
```diff
@@ -407,7 +407,7 @@
       post(
         api_v1_post_vote_path("999_999_999"),
         params: {
-          poll_answer_id: @poll_answer.id,
+          poll_answer: @poll_answer.id,
```
Comment (diaspora-code-review) [inline spec/integration/api/post_interactions_controller_spec.rb]: Layout/AlignHash: Align the elements of a hash literal if they span more than one line.

File: spec/integration/api/post_interactions_controller_spec.rb [hunk 6]
```diff
@@ -418,7 +418,7 @@
       post(
         api_v1_post_vote_path(@poll_post.guid),
         params: {
-          poll_answer_id: @poll_answer.id,
+          poll_answer: @poll_answer.id,
```
Comment (diaspora-code-review) [inline spec/integration/api/post_interactions_controller_spec.rb]: Layout/AlignHash: Align the elements of a hash literal if they span more than one line.

File: spec/integration/api/post_interactions_controller_spec.rb [hunk 7]
```diff
@@ -429,7 +429,7 @@
       post(
         api_v1_post_vote_path(@shared_post.guid),
         params: {
-          poll_answer_id: @poll_answer.id,
+          poll_answer: @poll_answer.id,
```
Comment (diaspora-code-review) [inline spec/integration/api/post_interactions_controller_spec.rb]: Layout/AlignHash: Align the elements of a hash literal if they span more than one line.

File: spec/integration/api/post_interactions_controller_spec.rb [hunk 8]
```diff
@@ -440,7 +440,7 @@
       post(
         api_v1_post_vote_path(@poll_post.guid),
         params: {
-          poll_answer_id: @poll_answer.id,
+          poll_answer: @poll_answer.id,
```
Comment (diaspora-code-review) [inline spec/integration/api/post_interactions_controller_spec.rb]: Layout/AlignHash: Align the elements of a hash literal if they span more than one line.

**Review by skyschub — APPROVED** [review]

**Pull request merged**